composer.json

```diff
@@ -20,7 +20,8 @@
         "symfony/templating": "^4.1"
     },
     "require-dev": {
-        "contao/core-bundle": "^4.6",
+        "contao/core-bundle": "^4.7",
+        "contao/news-bundle": "^4.7",
```

Comment:
Why do we need the news-bundle?

Reply:
Because it is instantiated by the AppKernel https://github.com/contao/contao/blob/master/core-bundle/tests/Functional/app/AppKernel.php#L48